test/unit/model/prediction/xgboost/test_xgb_quantile.py

```diff
@@ -0,0 +1,66 @@
+# SPDX-FileCopyrightText: 2017-2021 Alliander N.V. <korte.termijn.prognoses@alliander.com> # noqa E501>
```

Comment:
This test seems to be in the wrong location? It should be `test/unit/model/test_xgb_quantile.py` given the current layout.